backend/controllers/bulkAppointmentsConfirm.js

```diff
@@ -125,7 +125,7 @@ const bulkAppointmentsConfirmFactory = (elite2Api, logError) => {
     } catch (error) {
       return renderError(req, res, error)
     }
-
+    console.log('Code cov test')
```

Comment:
Come on mate, you know better than to commit console.logs

Reply:
:astonished:

Reply:
Have you considered trying to simplify it? `console.log(console.log==window.console.log) `

Reply:
#bantz